src/js/components/tablero/UnarchivePopout.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import ReactLoading from 'react-loading';
+import { loadColors } from '../../../assets/dataFormat/dataLoading';
+
+function getRandomInt(min, max) {
+  return Math.floor(Math.random() * (max - min)) + min;
+}
+const Numbers = () => {
+  const colors = ['green', 'yellow', 'red'];
+  const classColor = `circle ${colors[getRandomInt(0, colors.length)]}`;
+
+  return (
+    <div className="value">
+      <div className={classColor} /> {getRandomInt(3, 95)}
```

Comment:
why are you displaying the random number here?

Reply:
@pmuchowski I think that's just because we haven't linked that particular component to the backend yet.

Reply:
When an indicator is archived we can get them back from the "Archived Indicator" popout, where we display a preview of each indicator, on which we don't display real data but an image corresponding to the type. This is for the 3 numbers RAG indicator: we display a random number in the preview.